README.md

```diff
@@ -38,45 +40,90 @@ This project is inspired by [Google vizier](https://static.googleusercontent.com
+It is assumed that objective function f(x) does not change in the course of a `Experiment`.
 
-Represents a single optimization run over a feasible space. Each Study contains a configuration describing the feasible space, as well as a set of Trials. It is assumed that objective function f(x) does not change in the course of a Study.
+In v1alpha1, `Experiment` is defined as a CRD `StudyJob` in Kubernetes.
+In v1alpha2, `Experiment` is defined as a CRD `Experiment`.
 
 ### Trial
 
-A Trial is a list of parameter values, x, that will lead to a single evaluation of f(x). A Trial can be “Completed”, which means that it has been evaluated and the objective value f(x) has been assigned to it, otherwise it is “Pending”.
-One trial corresponds to one job, and the job kind can be [k8s Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/), [TFJob](https://www.kubeflow.org/docs/guides/components/tftraining/) or [PyTorchJob](https://www.kubeflow.org/docs/guides/components/pytorch/), which depends on the Study's worker kind.
+A `Trial` is a list of parameter values, x, that will lead to a single evaluation of f(x). A Trial can be “Completed”, which means that it has been evaluated and the objective value f(x) has been assigned to it, otherwise it is “Pending”.
+
+In v1alpha1, `Trial` is just a concept inside Katib and not exposed to users.
+In v1alpha2, `Trial` is defined as a CRD `Trial` in Kubernetes.
+
+### Job
+
+`Job` is also known as `Worker` in [Google Vizier][3].
+A `Job` refers to a process responsible for evaluating a Pending `Trial` and calculating its objective value.
+
+The job kind can be [Kubernetes Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/), [Kubeflow TFJob](https://www.kubeflow.org/docs/guides/components/tftraining/) or [Kubeflow PyTorchJob](https://www.kubeflow.org/docs/guides/components/pytorch/).
+Thus Katib supports multiple frameworks with the help of different job kinds.
 
 ### Suggestion
 
-A Suggestion is an algorithm to construct a parameter set. Currently Katib supports the following exploration algorithms:
+A Suggestion is an algorithm to construct a parameter set according to the `Experiment`. Then `Trial` will be created to evaluate the parameter set.
+
+Currently Katib supports the following exploration algorithms in v1alpha1:
 
-* random
-* grid
+* random search
+* grid search
 * [hyperband](https://arxiv.org/pdf/1603.06560.pdf)
 * [bayesian optimization](https://arxiv.org/pdf/1012.2599.pdf)
+* NAS based on reinforcement learning. The algorithm follows the idea proposed in [*Neural Architecture Search with Reinforcement Learning*](https://arxiv.org/abs/1611.01578), and the implementation is based on the open source implementation of [*Efﬁcient Neural Architecture Search via Parameter Sharing*](https://github.com/melodyguan/enas) (Also known as ENAS).
```

Comment:
Sorry, I should have pointed just now more clear. Our nasrl misses some feature (saying layer skip) of enas.

Reply:
Yeah, it is not ENAS, it is actually the original paper `Neural Architecture Search with Reinforcement Learning`. We noticed it here.

Reply:
in our implementation the numLayers is fixed, the original paper has no this limitation